docs/contracts.rst

```diff
@@ -469,9 +469,18 @@ View Functions
 
 Functions can be declared ``view`` in which case they promise not to modify the state.
 
+The following statements are modifying the state:
+
+#. Writing to state variables.
+#. :ref:`Creating other contracts <creating-contracts>`
+#. Sending Ether via calls.
+#. Using ``selfdestruct``.
+
```

Comment:
Calling functions that are not marked `view` or `pure`.

Reply:
Using low-level calls.
Using inline assembly that contains certain opcodes?

Reply:
Actually, do we want to mark modifiers as view and pure?

Reply:
Calling functions was a remark below the list, but moved it into.

Reply:
There is a separate issue (#1402) about having view/pure modifiers, I assume at this point we infer it.